packages/room/stream/stream.js

```diff
@@ -52,7 +81,9 @@ export const createStream = () => {
         origin: stream.origin,
         source: stream.source,
         mediaStream: stream.mediaStream,
+        audioLevel: 0,
         replaceTrack: stream.replaceTrack,
+        addVoiceActivity: stream.addVoiceActivity,
```

Comment:
addVoiceActivity is not a public method. It's a private method that will trigger the voice activity event on the stream. AudioLevel is a public property. So it will need a doc.

Reply:
I see, `addVoiceActivity()` is not a public method. The current issue with JavaScript is, we cannot create a public method that only available for internal module. When `addVoiceActivity()` is exposed as public method to be used in the peer module, it will also available for users to use the `addVoiceActivity()` method.

I'm still thinking about the approach, but probably we can use internal custom event for this kind of logic without making a public method. I think for current implementation this is fine, I will try to improve the flow when I start the refactoring flow.